### ISSUE TITLE
asyncio.ProactorEventLoop is not recognized when running on Windows

### ISSUE THREAD

**Issue by gordonwatts** [microsoft/pyright#381]
**Describe the bug**
pyright doesn't recognize `asyncio.ProactorEventLoop` on the Windows platform, giving the error "'ProactorEventLoop' is not a known member of module".

**To Reproduce**
The following code will show the error:
```
import asyncio
asyncio.ProactorEventLoop()
```

**Expected behavior**
This is a valid call on the `nt` platform - so it should not send up an error.

**VS Code extension or command-line**
Using VS Code, 1.1.7

**Additional context**
When it comes time to getting this to run on other platforms, this line is protected by an if statement (`if os.name == 'nt'`). Is that the appropriate way to protect this line from pyright's POV?

**Comment by erictraut** [comment]
Pyright allows you to specify the platform you are targeting. To do so, add the following line to your pyrightconfig.json file:

```
    "pythonPlatform": "Windows"
```
In investigating your issue, I just found a bug. Pyright isn't correctly translating the "Windows" string in the configuration to "win32", which is the string that is returned by sys.platform. I'll fix this in the next version of pyright. In the meantime, you can work around it by changing your pyrightconfig.json to:

```
    "pythonPlatform": "win32"
```
Note that this will need to be changed back to "Windows" once I publish the bug fix.

**Comment by gordonwatts** [comment]
Thanks! And thanks so much for taking care of pyright! I've found is very useful!

**Comment by erictraut** [comment]
This is now fixed in 1.1.8, which I just published.
